docs/content/integrating-sns.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Integrating OpenCensus with sns"
+date: "2022-04-06"
+type: page
+menu:
+  main:
+    parent: "Integrations"
+---
+
+Integration with Aws sns using the following methods will:
+
+## sns
+
+To add OpenCensus support for sns Load in OpencensusProvider.php file
+
+```php
+<?php
+
+            \OpenCensus\Trace\Integrations\Sns::load();
+
+```
+
+To add Spans
+
+```php
+<?php
+use Aws\Sqs\SqsClient;
+
+
+        ...
+        Tracer::inSpan(['name' => 'SNS:example'], function () {
```

Comment:
Need this wrapper line?

Reply:
yes, this is to tell client ops should be under a span